tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
```

Comment:
And calling wait_idle seems silly when there is a check_screen immediately after, so openQA is going to take it's time before moving onto it's next actions evaluating the check_screen anyway